packages/components/Checkbox/src/Checkbox.tsx

```diff
@@ -25,6 +25,7 @@ export const Checkbox = compose<ICheckboxType>({
   usePrepareProps: (userProps: ICheckboxProps, useStyling: IUseComposeStyling<ICheckboxType>) => {
     const defaultComponentRef = React.useRef(null);
     const {
+      accessibilityLabel,
```

Comment:
Do we have a plan of action for letting partners know to start transitioning from ariaLabel to accessibilityLabel?

Reply:
Nevermind, I do see the @deprecated tag in the comments of ariaLabel :)